blockchain/blockchain-wallet-v4-frontend
# fix(Login): Remove whitespace characters from Wallet ID during login.

fix #WEB4-1883

## Description
Remove whitespace characters from Wallet ID during login.

## Change Type
- Bug Fix

## Testing Steps
1.  When logging in, type in any Wallet ID (e.g., `2a507510-a7a2-437c-87e3-d9191a1a0603`).
2.  Attempt to add a space at the end of it (or anywhere else within the Wallet ID field).
3.  Notice the space isn't accepted.

## Code Checklist
